tests/ActivityTest.php

```diff
@@ -44,24 +44,21 @@ public function testInstantiation() {
 
     public function testFromJSONInvalidNull() {
         $this->setExpectedException(
-            'InvalidArgumentException',
-            'Invalid JSON: ' . JSON_ERROR_NONE
+            'TinCan\JSONParseErrorException'
         );
         $obj = Activity::fromJSON(null);
     }
 
     public function testFromJSONInvalidEmptyString() {
         $this->setExpectedException(
-            'InvalidArgumentException',
-            'Invalid JSON: ' . JSON_ERROR_NONE
+            'TinCan\JSONParseErrorException'
         );
         $obj = Activity::fromJSON('');
     }
 
     public function testFromJSONInvalidMalformed() {
         $this->setExpectedException(
-            'InvalidArgumentException',
-            'Invalid JSON: ' . JSON_ERROR_SYNTAX
+            'TinCan\JSONParseErrorException'
```

Comment:
Heh, on further inspection these tests weren't doing what they were originally thought to do anyways. `JSON_ERROR_NONE` actually ends up being '' which doesn't change the error message which is why they were passing to begin with.